cli/tests/pcluster/config/test_validators.py

```diff
@@ -524,7 +524,7 @@ def test_raid_validators(mocker, section_dict, expected_message):
 @pytest.mark.parametrize(
     "kms_key_id, expected_message",
     [
-        ("9e8a129be-0e46-459d-865b-3a5bf974a22k", None,),
+        ("9e8a129be-0e46-459d-865b-3a5bf974a22k", None),
```

Comment:
Why the linter was not able to detect this error before?
Did we forget to enable something in tox?

Reply:
probably it's just a newer version of black that implemented this